website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -0,0 +1,118 @@
+---
+id: versioning-releases
+title:  ORCA Versioning and Releases
+description: Provides information to developers on semantic versioning and the release process.
+---
+
+Much of this documentation is also found at [Cumulus](https://github.com/nasa/cumulus/blob/master/docs/development/release.md).
+
+## Versioning
+
+The ORCA team uses semantic versioning. More information about semantic
+versioning can be found [here](https://semver.org/).
+
+## Release Process
+
+1. **Create a release branch.**

```

Comment:
Wondering if these should be H3 headers instead of numbers so:
`### Create a release branch` instead of ` 1. **Create a release branch.**`. That or we provide the proper spacing so the paragraphs are a part of the list instead of separate when being displayed. This would be for list numbers 1 - 4. Thoughts?

Reply:
Made these into H3 headers.